src/spyglass/utils/dj_helper_fn.py

```diff
@@ -750,3 +816,16 @@ def _replace_nan_with_default(data_dict, default_value=-1.0):
             result[key] = default_value
 
     return result
+
+
+# Temporary log table for errors encountered during file edits, remove before merge (?)
+schema = dj.schema("sambray_export_error_log")
+
+
+@schema
+class ExportErrorLog(dj.Manual):
```

Comment:
If you plan to keep, I might structure this like ActivityLog where the insert triggers the user warning